File: Note-It/app/src/main/java/com/example/noteit/MainActivity.java
package com.example.noteit;

import static android.provider.AlarmClock.EXTRA_MESSAGE;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ArrayAdapter;
import android.widget.Button;
import android.widget.ListView;
import android.widget.Toast;

import java.io.File;
import java.nio.file.Files;
import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    public static final String EXTRA_MESSAGE = "com.example.noteit.MESSAGE";
    Button addNote;
    static ArrayAdapter adapter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        File files = getFilesDir();
        String[] fileArray = files.list();

        ArrayList<String> arrayList = new ArrayList<>();

        adapter = new ArrayAdapter(this, android.R.layout.simple_list_item_1, arrayList);

        for(String filename : fileArray){

            filename = filename.replace(".txt", "");
            System.out.println(filename);
            adapter.add(filename);
        }

        ListView listview = (ListView) findViewById(R.id.listview);
        listview.setAdapter(adapter);

        addNote = findViewById(R.id.addNote);


        listview.setOnItemClickListener(new AdapterView.OnItemClickListener() {
            @Override
            public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
                String item = listview.getItemAtPosition(i).toString();
                Intent intent2 = new Intent(getApplicationContext(), ShowNotesActivity.class);
                intent2.putExtra(EXTRA_MESSAGE, item);
                startActivity(intent2);
            }
        });
    }

    public void launchCreateActivity(View view) {

        Intent intent = new Intent(this, CreateNoteActivity.class);
        startActivity(intent);
    }
}

File: Firebase/AuthenticationWithFirebase/app/src/main/java/com/example/authenticationwithfirebase/MainActivity.kt
package com.example.authenticationwithfirebase

import android.content.Intent
import android.os.Bundle
import android.widget.ArrayAdapter
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.example.authenticationwithfirebase.R
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        button.setOnClickListener {

            if (mobileNumber.text.count() == 10) {
                val i = Intent(this, OTPValidation::class.java).apply {
                    putExtra("mobileNumber", mobileNumber.text.toString())
                }
                startActivity(i)
            } else {
                Toast.makeText(this, "Enter 10 digits mobile number",Toast.LENGTH_LONG).show()
            }
        }
    }
}